index.md

```diff
@@ -1,51 +1,61 @@
-- Using [IBM Cloud Functions](https://cloud.ibm.com/functions/learn/concepts), HRI exposes a Serverless RESTful Management API that is used to control and configure the system
-- [ElasticSearch](https://github.com/elastic/elasticsearch) is the distributed NoSQL data store that is used to store information about batches
+**Key technologies**
 
-## IBM Cloud Dependencies  
-The HRI was developed on the IBM cloud and currently does not support running on other public or private clouds. However, as a part of Alvearie, the HRI is being transitioned into an open source project, and the goal is to support public and private cloud deployments. The latest version has been pushed to GitHub, and the team continues to work on moving all development activities into the open and making the HRI support other clouds. Please see the [Roadmap](roadmap.md) for additional details.   
+- [**Event Streams**](glossary.md#event-streams): This is an IBM Cloud-based [Apache Kafka](https://kafka.apache.org/) managed service, the technology used for producing and consuming the data streams.
+- [**IBM Cloud Functions**](glossary.md#ibm-cloud-functions): Using these, Health Record Ingestion exposes a serverless RESTful Management API that is used to control and configure the system. 
+- [**Elasticsearch**](glossary.md#elasticSearch): This is the distributed NoSQL data store that is used to store information about batches.
 
-## Core Architecture
+## IBM Cloud dependencies  
+Health Record Ingestion was developed on the IBM Cloud and currently does not support running on other public or private clouds. However, as part of Project Alvearie, Health Record Ingestion is being transitioned into an open source project, with a goal of supporting public and private cloud deployments. The latest version has been pushed to GitHub. The team continues to work on moving all development activities into the open, and making Health Record Ingestion support other clouds. For more information about future plans, see the [Roadmap](roadmap.md).   
+
+## Core architecture
+<br>
+**Figure: Components of the Health Record Ingestion service**
+<br><br>
 ![core-architecture](assets/img/architecture-core.png)
+<br><br>
 
 ### Topics
-Health data, which may include [PHI](glossary.md#phi), is written to and read from the Kafka (IBM Event Streams) topics. There **_must be_** _separate topics for each tenant and Data Integrator_ in order to meet data separability requirements. 
+Kafka (IBM Event Streams) stores streams of records in categories or feeds called **topics**. Health data, which can include [Protected Health Information (PHI)](glossary.md#protected-health-information-phi), is written to and read from topics. To meet data separability requirements, **there must be separate topics for each tenant and Data Integrator**. 
```

Comment:
I don't think we should try to define Kafka 'topics' (first sentence). That just invites people to disagree.  Rather just include links, so they can go learn if they don't know what they are.